src/utils/error_vector.C

```diff
@@ -318,4 +318,108 @@ void ErrorVector::plot_error(const std::string & filename,
     }
 }
 
+
+void ErrorVector::plot_smoothness(const std::string & filename,
+                             const MeshBase & oldmesh) const
```

Comment:
This is basically just `plot_error()` but with a different variable name in the output file?  Let's implement that with an optional string argument to `plot_error()`, not with a hundred lines of copy-and-paste.